frontend/docs/pages/home/lifespans.mdx

```diff
@@ -0,0 +1,36 @@
+import { Callout, Card, Cards, Steps, Tabs } from "nextra/components";
+import UniversalTabs from "@/components/UniversalTabs";
+import { GithubSnippet, getSnippets } from "@/components/code";
+
+export const Lifespans = {
+  path: "examples/lifespans/worker.py",
+};
+
+export const getStaticProps = ({}) => getSnippets([Lifespans]);
+
+# Lifespans
```

Comment:
Let's add a note that this feature is experimental and signatures may change

Reply:
Good call